.github/workflows/secrets.test-workspaces.yml

```
@@ -15,14 +15,16 @@ jobs:
     permissions:
       contents: read
+      actions: read
       statuses: write
     steps:
       - uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10
         with:
           ref: ${{ github.event.workflow_run.head_sha }}
+          fetch-depth: 0
           persist-credentials: false
       - name: Set pending status
         uses: ./.github/workflows/shared-actions/set-pr-status
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/workflows/shared-actions/set-pr-status": line 5: unexpected key "type" for definition of input "github_token"

(action)